org.omg.sysml/src/org/omg/sysml/adapter/FlowConnectionUsageAdapter.java

```diff
@@ -40,23 +42,47 @@ public FlowConnectionUsage getTarget() {
 		return (FlowConnectionUsage)super.getTarget();
 	}
 	
+	// From 
```

Comment:
Oops. I decided to put comments inside the body of the `addDefaultGeneralType` method instead of before it, but then forgot to delete this incomplete comment. It is gone now.